### ISSUE TITLE
Fix `function-no-unknown` false positives for `anchor` and `anchor-size`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7566]
### What minimal example or steps are needed to reproduce the bug?

```css
.bar { foo: anchor(); }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "function-no-unknown": true
  }
}
```

### How did you run Stylelint?

https://stylelint.io/demo

### references

https://www.w3.org/TR/css-anchor-position-1/
https://caniuse.com/css-anchor-positioning
https://github.com/oddbird/css-anchor-positioning

### What did you expect to happen?

no error

### What actually happened?

`Unexpected unknown function "anchor"`
`Unexpected unknown function "anchor-size"`

### Do you have a proposal to fix the bug?

either add exceptions or update https://github.com/niksy/css-functions-list
